tax_helper.rb

```diff
@@ -0,0 +1,34 @@
+module TaxHelper
+
+  LIST_OF_EXEMPT_ITEMS = %w(food medicin chocolates pills book books chocolate )
+
+# to check is item exempt from tax
+  def exempt_good?(str)
+    item  = str.split(" ")
+    exempted_item = item & LIST_OF_EXEMPT_ITEMS
+   !exempted_item.empty?
```

Comment:
```suggestion
   exempted_item.present?
```
Should work?

Reply:
ma'am I tried using present but it is not working !!

Reply:
ohh okay, `present?` is a Rails method. Ignore this comment.